.mailmap

```diff
@@ -562,11 +574,15 @@ Peng Cheng Zhang <pengcheng.zhang@easystack.cn>
 Peng Cheng Zhang <pengcheng.zhang@easystack.cn> <pc@localhost.localdomain>
 Peng Lai <penglaiyxy@gmail.com>
 Peng Lai <penglaiyxy@gmail.com> <penglaiyxy>
+Pere Diaz Bou <30913090+pereman2@users.noreply.github.com>
```

Comment:
This email address seems to be already accounted for two lines below?

Reply:
ACK